torch/onnx/symbolic_opset9.py

```diff
@@ -1668,7 +1668,13 @@ def conv_tbc(g, input, weight, bias, pad):
     if sym_help._operator_export_type == torch.onnx.OperatorExportTypes.ONNX_ATEN_FALLBACK:
         return g.op("ATen", input, weight, bias, operator_s="conv_tbc", pad_i=pad)
     else:
-        return sym_help._onnx_unsupported('conv_tbc')
+        # input = (time, batch, in_channels)
```

Comment:
Is this always the case that input has 3 dimensions?

Reply:
Yes, see https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/ConvolutionTBC.cpp#L8-L10